### DISCUSSION TITLE
API for custom model <> token usage pricing

### DISCUSSION THREAD

**Discussion by rydamckinney** [langfuse/langfuse#1707] [Ideas]
### Describe the feature or potential improvement

Requesting a POST (and GET to check current custom model definitions) API route for adding custom internal model pricing. Ideally an `externalId` can be provided to link the definition in langfuse to internal model db records

**Comment by marcklingen** [comment]
Turning this into an issue as it is a simple addition and super valuable when fine-tuning models on a regular schedule

**Reply by marcklingen** [reply]
https://github.com/langfuse/langfuse/issues/1708

**Comment by marcklingen** [comment]
@rydamckinney, added in [https://github.com/langfuse/langfuse/pull/2531](https://github.com/langfuse/langfuse/pull/2531) and released in [https://github.com/langfuse/langfuse/releases/tag/v2.59.0](https://github.com/langfuse/langfuse/releases/tag/v2.59.0)
